src/lib.rs

```diff
@@ -581,6 +607,8 @@ mod test {
     #[test]
     fn exclude_relative_path() {
         let watch = Watch {
+            shell_commands: Vec::new(),
+            cargo_commands: vec!["check".to_string()],
```

Comment:
not sure why you need this for that particular test that test the exclude relative path thingy